File: src/test/testData/java/implicit/Combined.java
package ru.ustinov;

import java.net.URL;
import java.util.*;

class Simple {
    public static void main() {
        LinkedList<URL> a = new LinkedList<>();
        HashSet<URL> b = new HashSet<>();
        Hashtable<URL, Integer> c = new Hashtable<>();
        HashMap<URL, Integer> d = new HashMap<>();
        ArrayList<URL> e = new ArrayList<>();
        Queue<URL> f = new ArrayDeque<>();
        a.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>();
        b.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>();
        c.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>();
        d.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>();
        e.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>();
        f.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>();
    }
}

File: src/test/testData/kotlin/implicit/Combined.kt
import java.net.URL
import java.util.HashMap
import java.util.HashSet
import java.util.Hashtable
import java.util.ArrayList
import java.util.LinkedList
import java.util.Queue
import kotlin.Array

@Suppress("UNUSED_VARIABLE")
fun main() {
    lateinit var a: Array<URL>
    lateinit var b: HashSet<URL>
    lateinit var c: HashMap<URL, Int>
    lateinit var d: Hashtable<URL, Int>
    lateinit var e: ArrayList<URL>
    lateinit var f: LinkedList<URL>
    lateinit var g: Queue<URL>
    a.<error descr="[MISSING_DEPENDENCY_SUPERCLASS] Cannot access 'java.io.Serializable' which is a supertype of 'kotlin.Array'. Check your module classpath for missing or conflicting dependencies"><warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning></error>()
    b.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>()
    c.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>()
    d.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>()
    e.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>()
    f.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>()
    g.<warning descr="Attempt to call hashCode() implicitly on DataStructure which contains URL">hashCode</warning>()
}